account_invoice_line_unit_price_no_tax/models/account_move_line.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+
+    _inherit = "account.move.line"
+
+    price_unit_no_taxes = fields.Float(
```

Comment:
This field shloud have the same precision as the original unit price field, don't you think ?
https://github.com/odoo/odoo/blob/15.0/addons/account/models/account_move.py#L3496